.travis.yml

```diff
@@ -13,13 +13,11 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  - SYMFONY_VERSION=2.7.*
+  - SYMFONY_VERSION=3.0.*
```

Comment:
`3.0.*` does not make sense, as that's unmaintained. The default should be to not force a Symfony version. And then having some jobs forcing LTS versions